feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.ts

```diff
@@ -66,4 +68,65 @@ export class ConfiguratorOverviewFormComponent {
       ) !== undefined
     );
   }
+
+  /**
+   * Verifies whether the next or the previous attributes are same.
+   *
+   * @param {Configurator.AttributeOverview[]} attributes - Attribute array
+   * @param {number} index - Index of the attribute in the array
+   * @return {boolean} - 'True' if it is the same attribute, otherwise 'false'
+   */
+  isSameAttribute(
```

Comment:
this is only used in `getStyleClasses`, so maybe it can be `protected`.

Reply:
This component was delivered. I don't think that it is a good idea to change a modifier to `protected`

Reply:
`isSameAttribute` is a new function, right?